examples/real-world/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
   }
   dependencies {
```

Comment:
you should probably remove this entire `buildscript` block from this `build.gradle` file. it is redundant.

Reply:
sure, but where to put `dependencies` ?